libraries/AP_OSD/AP_OSD.h

```diff
@@ -148,6 +148,10 @@ class AP_OSD_Screen : public AP_OSD_AbstractScreen
     AP_OSD_Setting bat_volt{true, 24, 1};
     AP_OSD_Setting rssi{true, 1, 1};
     AP_OSD_Setting link_quality{false,1,1};
+    AP_OSD_Setting crsf_rf_mode{false, 1, 1};
```

Comment:
I don't think these need to be CRSF specific apart from perhaps mode

Reply:
ok, fixed this